ts/src/bingx.ts

```diff
@@ -1683,10 +1683,10 @@ export default class bingx extends Exchange {
             'datetime': this.iso8601 (timestamp),
             'lastTradeTimestamp': lastTradeTimestamp,
             'symbol': symbol,
-            'type': type,
+            'type': type.toLowerCase(),
             'timeInForce': undefined,
             'postOnly': undefined,
-            'side': side,
+            'side': side.toLowerCase(),
```

Comment:
@ZetZ1 this should not be needed we're already parsing them as lowercase strings

```
        const side = this.safeStringLower (order, 'side');
        const type = this.safeStringLower (order, 'type');
```

Reply:
@ZetZ1 no worries, feel free to open a new one if you find something weird